pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "bson"
+version = "0.5.10"
+authors = [
+    { name = "Ayun Park", email = "iamparkayun@gmail.com" }
+]
+description = "BSON codec for Python"
+readme = "README.rst"
+requires-python = ">=3.9"
+dependencies = [
+    "python-dateutil>=2.9.0.post0",
+]
+license = {text = "BSD"}
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+keywords = ["BSON", "codec"]
+
+[project.urls]
+Homepage = "https://github.com/py-bson/bson"
+PyPI = "https://pypi.org/project/bson/"
+
+[dependency-groups]
+lint = [
+    "ruff>=0.8.3",
+]
+test = [
+    "pytest>=8.3.4",
+]
+
+[tool.ruff]
+line-length = 80
+
+[tool.ruff.lint]
+select = ["E", "F", "I", "FA"]
+ignore = ["B904", "B028", "F841", "E741"]
+
+[tool.ruff.format]
+skip-magic-trailing-comma = true
+
+[tool.ruff.lint.isort]
+combine-as-imports = true
+split-on-trailing-comma = false
+
+[tool.ruff.lint.per-file-ignores]
+"__init__.py" = ["F401"]
+
+[build-system]
```

Comment:
This section is traditionally put on top -- which I like, so I know what's going on right away.